assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -80,12 +84,101 @@ export const actions = {
 			type: RECEIVE_ACCOUNTS_PROPERTIES_PROFILES_FAILED,
 		};
 	},
+
+	/**
+	 * Creates a new Analytics account.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} args              Argument params.
+	 * @param {string} args.accountName  Google Analytics account name.
+	 * @param {string} args.propertyName Google Analytics property name.
+	 * @param {string} args.profileName  Google Analytics profile name.
+	 * @param {string} args.timezone     Google Analytics timezone.
+	 * @return {Function} Generator function action.
+	 */
+	*createAccount( { accountName, propertyName, profileName, timezone } ) {
+		invariant( accountName, 'accountName is required.' );
+		invariant( propertyName, 'propertyName is required.' );
+		invariant( profileName, 'profileName is required.' );
+		invariant( timezone, 'timezone is required.' );
+
+		try {
+			const accountTicket = yield actions.fetchCreateAccount( { accountName, propertyName, profileName, timezone } );
+			yield {
+				type: RECEIVE_CREATE_ACCOUNT,
+			};
+
+			return actions.receiveCreateAccount( { accountTicket } );
+		} catch ( error ) {
+			// TODO: Implement an error handler store or some kind of centralized
+			// place for error dispatch...
+			return actions.receiveCreateAccountFailed( { accountName, error } );
+		}
+	},
+
+	*fetchCreateAccount( { accountName, propertyName, profileName, timezone } ) {
+		yield {
+			type: CREATE_ACCOUNT_STARTED,
+		};
+
+		return {
+			payload: { accountName, propertyName, profileName, timezone },
+			type: FETCH_CREATE_ACCOUNT,
+		};
+	},
+
+	/**
+	 * Adds a account ticket to process.
+	 *
+	 * Adds the newly-created account ticket to the data store.
+	 *
+	 * @since n.e.x.t
+	 * @private
+	 *
+	 * @param {Object} args               Argument params.
+	 * @param {Object} args.accountTicket Google Analytics create account ticket object.
+	 */
```

Comment:
Just FYI, we've decided to not document all these internal `fetch...`/`receive...` etc actions, controls, etc. No need to remove these docs from here, but wanted to let you know for the future. So from the new actions here, theoretically only `createAccount` needs to have docs.

Reply:
👍